custom_components/spotcast/cast.py

```diff
@@ -0,0 +1,55 @@
+from collections.abc import Callable
+
+from homeassistant.components import spotify as ha_spotify
+from homeassistant.components.media_player import BrowseMedia
+from homeassistant.components.media_player.const import MEDIA_CLASS_APP
+import homeassistant.core as ha_core
+from pychromecast import Chromecast
+
+
```

Comment:
What will use that logger?

Reply:
Yes, it is in reference to this comment.

I don't want us to be too complex in the debug logs we put. It's really for debugging when a user will eventually bring an issue regarding this workflow.

We could simply add a log stating that Spotcast is attempting X. Since we do get variables for `async_browse_media` and `async_play_media`, I would add this information in the log.

We should also check if we get scenarios where we run to the end of a function without returning properly something. If that's the case, we could check the behavior of Home assistant and check if an error log could provide more information.

Reply:
OK, got it 👍

Reply:
Debug prints added in 5d8f4c7